react-design/my-app/src/AccordionButtons.tsx

```diff
@@ -0,0 +1,29 @@
+type AccordionButtonsProp = {
+  current: number;
+  onClick: (index: number) => void;
+};
+
+export function AccordionButtons({ current, onClick }: AccordionButtonsProp) {
```

Comment:
It would be best to pass in the number of buttons so that it will work no matter how many items there are.